uint16 j1_vrx
uint16 j1_vry
bool j1_button

uint16 j2_vrx
uint16 j2_vry
bool j2_button

uint16 j3_vrx
uint16 j3_vry
bool j3_button
